WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt

```diff
@@ -84,8 +86,15 @@ class PagesViewModel
                 delay(200)
                 searchJob = null
                 if (isActive) {
-                    val result = pageStore.search(site, searchQuery)
-                            .map { Page(it.pageId.toLong(), it.title, null) }
+                    val result = pageStore.groupedSearch(site, searchQuery)
+                            .map { (status, results) ->
+                                listOf(Divider(1, status.toResource())) +
```

Comment:
Is the `Id` property necessary in the `Divider` class?

Reply:
it is but it shouldn't be 1! :-) we should be using a hashcode to identify divider instead